packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -9,6 +9,14 @@ import OrderDetailsFooter from 'Components/order-details/order-details-footer.js
 const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
     const { order_store, sendbird_store } = useStores();
 
+    const showMobileOrderFooter = () => {
+        const footer = () => <OrderDetailsFooter order_information={order_store.order_information} />;
+        if (order_store.order_information.should_show_order_footer) {
+            return footer;
```

Comment:
Can we return the JSX tag here? rather than an anonymous function?

Reply:
The `MobileFullPageModal` component prop types expects a function, it would return an error if a component is passed in:
![Screenshot 2022-04-25 at 1 57 49 PM](https://user-images.githubusercontent.com/103016120/165028992-4cfb1595-d906-4bdb-be08-ec69a9270d0f.png)
![Screenshot 2022-04-25 at 1 58 12 PM](https://user-images.githubusercontent.com/103016120/165029035-5209bcd0-a336-431d-b78b-0d896847f220.png)
<img width="589" alt="Screenshot 2022-04-25 at 1 59 01 PM" src="https://user-images.githubusercontent.com/103016120/165029126-f9aafd86-752d-48f0-ba7c-617383fd1c73.png">